### ISSUE TITLE
ClrAccordion panel service has a race condition that prevents opening of other panels when default strategy used

### ISSUE THREAD

**Issue by hippee-lee** [vmware/clarity#5822]
## Describe the bug
ClrAccordion has a race condition in the panel service. 

### Symptom #1
When a consumer wants to set an Accordion open on initilaization and it is **not** using the clrAccordionMultiPanel strategy they should be able to set two way binding on the `clr-accordion-panel` element like so: `[(clrAccordionPanelOpen)]="sections.section1"`. This results in an infinite loop and the panel does not open.

### Symptom #2
When a consumer uses one way binding `[clrAccordionPanelOpen]="sections.section1"` to open a panel with an accordion using clrAccordionMultiPanel strategy the panel should open. Rigth now it does not open.

### Symptom #3
When the consumer is using `clrAccordionMultiPanel` and they programmatically open a closed panel it should open. Right now it does not open. An example of this is a button inside the open panel #2 that can toggle the `[clrAccordionPanelOpen]="sections.section1"` property of panel #1. Right now the behavior closes panel #2 and does not open panel #1. 

## How to reproduce

https://stackblitz.com/edit/clarity-v4-light-theme-zqdcqi?file=src/app/app.component.ts

In the example notice that:

1. panel #1 is not opened after initialization.
2. when panel two is opened manually and the `Open Section 1` button is opened is clicked, panel #1603 
3. If two way binding is added to `[(clrAccordionPanelOpen)]="sections.section1"` the result is an infinite loop where the value of `sections.section1` is continually flipped between true <-> false.

## Expected behavior

1. When a ClrAccordion is using the default (single) clrAccordionMultiPanel strategy, panels with `[clrAccordionPanelOpen]="sections.section1"` should be open after initialization
2. When a ClrAccordion is using the default (single) clrAccordionMultiPanel strategy, and a button inside a sibling panel is used to toggle a different panel, the open panel should close and the other panel should open
3. When two way binding is used with: clrAccordionPanelOpen there should not be an infinate loop

## Versions

**Clarity project:**

- [ ] Clarity Core
- [x] Clarity Angular/UI

**Clarity version:**

- [ ] v3.x
- [ ] v4.x
- [x] v5.x

**Framework:**

- [x] Angular
- [ ] React
- [ ] Vue
- [ ] Other:

**Framework version:**
_ie: Angular 11

**Device:**

- Type: [e.g. MacBook]

## Additional notes

_Add any other notes about the problem here._

**Comment by hippee-lee** [comment]
I didn't get a fix for this yet. It is a lifecycle issue that is caused by the Observable implementation in the model. Not sure if there is a way to address it w/o removing the toggle function from the modal so it can be controlled with the lifecycle instead of reacting to the changes. I tried guarding against first run states and useing an oompa loompa but they caused propagating changes

**Comment by bdryanovski** [comment]
I notice the same thing last week when I was investigating another issue related to the Stepper and Accordion - Both of them use the same service in their roots.

But when changed the Observable to Subject instead of Behavior, I got many errors with the lifecycles. 

So I did some work on that but did not find a super clean solution - just patching the patch - probably need a lot more reworking from the ground up.
